docs/quickstart/html.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Redoc HTML element
+---
+
+# ReDoc HTML element
+
+## TL;DR Final code example
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>ReDoc</title>
+    <!-- needed for adaptive design -->
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
+
+    <!--
+    ReDoc doesn't change outer page styles
+    -->
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <redoc spec-url='http://petstore.swagger.io/v2/swagger.json'></redoc>
+    <script src="https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js"> </script>
+  </body>
+</html>
+
+```
+
+:::attention Running Redoc locally requires an HTTP server
+Loading local OpenAPI specifications is impossible without running a web-server because of issues with
+[same-origin policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) and
+other security reasons. 
+:::
+
+### Running ReDoc locally
+
+If you want to view your ReDoc output locally, you can simulate an HTTP server.
```

Comment:
How about `preview-docs`? That runs a local server too. Should we add it as another option? https://redoc.ly/docs/cli/commands/preview-docs/

Reply:
Good idea. I have made it the first option.